src/components/SearchForm/FuzzySearch.js

```diff
@@ -0,0 +1,119 @@
+import React, { PureComponent } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import search from 'websoc-fuzzy-search';
+import { updateFormValue, resetFormValues } from '../../actions/RightPaneActions';
+
+const emojiMap = {
+    GE_CATEGORY: '🏫', // U+1F3EB :school:
+    DEPARTMENT: '🏢', // U+1F3E2 :office:
+    COURSE: '📚', // U+1F4DA :books:
+    INSTRUCTOR: '🧑', // U+1F9D1 :adult:
```

Comment:
Let's use the teacher emoji
```suggestion
    INSTRUCTOR: '🧑‍🏫',
```

Reply:
Slight issue with this: the zero-width joiner doesn't actually render on my laptop, so the teacher "emoji" actually displays as two separate emojis, like so:
![image](https://user-images.githubusercontent.com/89349085/163302186-0eafe2c6-0eae-4188-95fe-b46b76c4f7d8.png)

Can you replicate this on your setup?

Reply:
Hmm are you on Windows or Linux? I'm on Mac

Reply:
I'm on Windows. Did some more experimentation, and it looks like Windows' emoji font spec only supports ZWJ on gendered person emojis: 👨‍🏫 (man + school -> male teacher) and 👩‍🏫 (woman + school -> female teacher) render as one emoji, but 🧑‍🏫 (person + school) doesn't.

Reply:
Classic windows

Reply:
Uh oh so we want to use the male or female one?

Reply:
Yeah unfortunately I think it's either choosing between them or using the person emoji 😕

Reply:
After some discussion on Discord Chase suggested to use 🍎 (apple), which sounds like a good enough idea to me, so it has been implemented in 13f7a30.